resources/views/auth/info.blade.php

```diff
@@ -0,0 +1,159 @@
+@extends('layouts.app')
+
+@section('content')
+    <div class="container">
+        <div class="row">
+            <div class="col-md-10 col-md-offset-1">
+                <div class="card card-default">
+                    <div class="card-header">@lang('user.personal_information'):</div>
+                    <div class="card-body">
+                        <form method="post" action="">
+                            @csrf
+                            <div class="form-group">
+                                <label class="col-sm-4 control" class="col-sm-4 control" for="name">@lang('user.name')</label>
+                                <div class="col-sm-8">
+                                <input class="form-control" type="text" name="name" id="name" required>
```

Comment:
@luksan47 Shouldn't we simply use a better describing name here?
Like `first_name` and `last_name`?

@erko185, I think we should have both `first_name` and `last_name` in one line (if fits), using Bootstrap, and only then wrap into two lines. Regards to the other fields, they look fine line by line.